di_website/common/blocks.py

```diff
@@ -332,6 +333,17 @@ class Meta:
         icon = 'fa-image'
 
 
+class FullWidthVideoBlock(StructBlock):
+    full_width = EmbedBlock(
```

Comment:
Why would you name it full width? Doesn't make sense... does it take in a full width?